src/vcastCommandRunner.ts

```diff
@@ -62,7 +62,9 @@ function processExceptionFromExecuteCommand(
 
   // 99 is a warning, like a mismatch opening the environment
   if (error && error.status == 99) {
-    commandStatus.stdout = error.stdout.toString();
+    let stdoutString =  error.stdout.toString();
+    // Needed because of the annoying version miss-match message from vcast
+    commandStatus.stdout = stdoutString.split ("ACTUAL-DATA", 2)[1].trim ()
```

Comment:
`prettier` wants to change this, *but* there's quite a few `prettier` changes against `src`.

I suggest we merge this and then do a `prettier` branch.

Reply:
I prettier'd this one file.